mods/dye/init.lua

```diff
@@ -20,13 +20,104 @@ dye.dyes = {
 	pink		= { name = "Pink", html = "#FF7FFF" },
 }
 
--- Define items
+----
+-- Helper functions.
+----
+
+--[[
+	Convert an html color in the format of #RRGGBB into red, green, and blue values on the range of [0..255].
+]]
+function hex2rgb(hex)
```

Comment:
I actually meant for this to be hex2rgb, rgb2hex, rgbToHsv, hsvToRgb, and darken to be local functions.  I can change these to local, then remove the camel casing.